main.py

```diff
@@ -105,18 +106,19 @@ def evaluate_policy(policy, env, episodes):
     Run the environment env using policy for episodes number of times.
     Return: average rewards per episode.
     """
-    total_reward = 0.0
+    rewards = []
     for _ in range(episodes):
-        state = env.reset()
+        state = np.float32(env.reset())
         is_terminal = False
         while not is_terminal:
             action = policy.get_action(tf.convert_to_tensor([state], dtype=tf.float32))
             # remove the batch_size dimension if batch_size == 1
             action = tf.squeeze(action, [0]).numpy()
             state, reward, is_terminal, _ = env.step(action)
-            total_reward += reward
+            state, reward = np.float32(state), np.float32(reward)

```

Comment:
Why are we modifying the evaluate_policy function? I was under the impression that we'd simply append evaluated rewards in the main function to some list

Reply:
It's for killing the warning of float32 and float 64. And I insist on returning a list. We also need std, apart from mean.